### DISCUSSION TITLE
Not getting options in custom plugin, also not able to see newsletter preset blocks

### DISCUSSION THREAD

**Discussion by jrodrguez08** [GrapesJS/grapesjs#3553] [Q&A]
Hi, I'm having some issues trying to load content using a custom storage manager as a plugin, that's the only way it works according this: https://github.com/artf/grapesjs/issues/2526

I need to pass the data to the load method, I'm trying to use the plugin options , but I get an empty object, also my custom storage plugin is making the newsltter preset to cause an error, not really sure why, I just want to get the custom storage working:

heres the error I get:
`grapes.min.js?a826:2 Uncaught TypeError: Cannot read property 'el' of undefined`

I created this sandbox for you guys to check the empty options issue that I mentioned
https://codesandbox.io/s/grapesjs-3ywko

any help would be awesome, great library by the way

**Comment by artf** [comment]
Please check this guide https://grapesjs.com/docs/modules/Plugins.html 
You're basically mixing a named plugin with a basic one. Indeed, it will work if you do this in your demo
```js
pluginsOpts: {
        [darkStorage]: { // instead of `darkStorage: {`
          test: "test"
        },
...
```
